.github/workflows/nucliadb_node.yml

```diff
@@ -33,14 +31,12 @@ on:
       - "Dockerfile.node_sidecar"
       - "Cargo.lock"
       - "Cargo.toml"
-      - "nucliadb_paragraphs_tantivy/**"
```

Comment:
We should add this folders on other worlflows were node moficiation triggers a build on the PR

Reply:
Im not entirely sure what do you mean by that, I think after [cd0ec62](https://github.com/nuclia/nucliadb/pull/539/commits/cd0ec62f84adaaf78c966759008166e7b88fc075) I have change all the pointers to `nucliadb_paragraphs_tantivy` to the current name `nucliadb_paragraphs`. Am I missing something?